packages/expo-localization/src/Localization.ts

```diff
@@ -51,6 +57,7 @@ export const isRTL = ExpoLocalization.isRTL;
 
 // @needsAudit
 /**
+ * @deprecated Use Localization.getPreferredLocales() instead.
```

Comment:
As discussed, let's not deprecate `locale` right now as it's commonly used in apps using i18n-js lib. In the future, when we remove module constants, we can make it a dynamic property.